src/include/security/dialogs.rb

```diff
@@ -568,6 +570,21 @@ def OverviewDialog
       deep_copy(ret)
     end
 
+    def vbox_boot_permissions
+      vbox = VBox(VSpacing(1), settings2widget("CONSOLE_SHUTDOWN"))
+
+      if @display_manager
```

Comment:
I'd prefer using `Empty()` widget trick instead of those `vbox <<` additions to be more consistent with the other code. See this [example](https://github.com/yast/yast-registration/blob/7465f820ccb9661828cbb57cb78f83e96c4ef74a/src/lib/registration/ui/import_certificate_dialog.rb#L97).